apps/web/pages/booking/[uid]/embed.tsx

```diff
@@ -1,8 +1,9 @@
 "use client";
```

Comment:
Legacy page does not need "use client" now, we import view directly to the future page.